stdlib/public/Concurrency/Task.swift

```diff
@@ -70,58 +86,58 @@ extension Task {
 @available(macOS 9999, iOS 9999, watchOS 9999, tvOS 9999, *)
 extension Task {
 
-  /// Returns the `current` task's priority.
+  /// The current task's priority.
   ///
-  /// If no current `Task` is available, returns `Priority.default`.
+  /// If you access this property outside of any task,
+  /// its value is `Priority.default`.
```

Comment:
FIXME: I thought we fixed this -- this should be `.unspecified` (and in implementation as well)